test/create_examples_emr_test.go

```diff
@@ -88,13 +95,20 @@ func TestCreateExamplesEmr(t *testing.T) {
 			})
 
 			defer test_structure.RunTestStage(t, "teardown", func() {
-				teraformOptions := test_structure.LoadTerraformOptions(t, tempTestFolder)
-				terraform.Destroy(t, teraformOptions)
+				terraformOptions := test_structure.LoadTerraformOptions(t, tempTestFolder)
+				terraformOptions.MaxRetries = 5
```

Comment:
What's the motivation for 5 retries? Are issues so intermittent that we need this many or is this to drive down the possibility?

With so many retries, we could have some very long tests (EMR clusters take ~7-15 minutes to start). If there is an actual issue, we may be spinning up / down resources for quite a while.

What if we brought the number of retries down to 2? If it didn't work once, maybe that's an intermittent issue, but if it didn't work twice, maybe it's real and we shouldn't retry anymore.